subscription/downgrade.md

```diff
@@ -7,7 +7,7 @@ redirect_from:
 - /docker-hub/billing/downgrade/
 ---
 
-You can downgrade your Docker subscription anytime. When you downgrade your subscription, changes are applied at the end of your billing cycle.
+You can downgrade your Docker subscription anytime. When you downgrade your subscription, changes are applied immediately, and you will no longer have access to the prior services.
 
 For example, if you are currently on a **Docker Team** subscription which is billed on the 8th of every month and you choose to downgrade to **Docker Personal** on the 15th, your plan will be active until 7th of the following month. You will be transferred to Docker Personal on the 8th of the following month.
```

Comment:
@jerae-duffin Should we update this example as well? Maybe use a Business subscription as an example?

Reply:
Updated